release/build_release.sh

```diff
@@ -121,6 +121,8 @@ function main {
 
 		lc_time_run generate_api_source_jar
 
+		lc_time_run generate_distro_jar
```

Comment:
@pgydev, is it really necessary since it's already being invoked inside the method `generate_poms`?

Reply:
@natocesarrego the function that being invoked in `generate_poms` is the `generate_pom_release_distro`